tools/xcodeproj_shims/installers/indexstores.sh

```diff
@@ -6,7 +6,7 @@ set -euo pipefail
 
 echo "Start remapping index files at `date`"
 
-FOUND_INDEXSTORES=`grep -o 'command_line: "\(.*\.indexstore\)' $BAZEL_BUILD_EVENT_TEXT_FILENAME | sed 's/.*command_line: "//' | sort | uniq` || true
+FOUND_INDEXSTORES=`grep -o 'command_line: "\(.*\.indexstore\)\|command_line: "\(.*_global_index_store\)' $BAZEL_BUILD_EVENT_TEXT_FILENAME | sed 's/.*command_line: "//' | sed 's/-Xwrapped-swift=-global-index-store-import-path=//' | sort | uniq` || true
```

Comment:
This regex was not picking up all stores we need, this should now pick the swift index stores and also the correct path to the global one where ObjC units/records are.